dune-project

```diff
@@ -1,359 +1,304 @@
 (lang dune 2.0)
-(formatting (enabled_for ocaml))
+
+(formatting
+ (enabled_for ocaml))
```

Comment:
Remove this or nobody else will try to keep the formatting

Reply:
If I remove this and run `make format`, it puts the libraries/modules/... on one line.

Reply:
How were the dune files formatted? It adds a lot of churn

Reply:
It probably is my vs code configuration. I'll try to remove the churn.